Havit.Blazor.Components.Web.Bootstrap/Layouts/HxListLayout.razor.css

```diff
@@ -1,5 +1,11 @@
+.hx-list-layout {
+	--hx-list-layout-header-spacer: .5rem;
```

Comment:
Is this a new CSS variable?
* The default value should be set in `defaults.lib.css`
* Should be added to documentation `HxListLayout_Documentation.razor`

cc @crdo

Reply:
@petrhank To align this with the rest of the codebase, it should be `--hx-list-layout-header-gap`, and it should be applied to the header instead of its children.